config/manager/manager.yaml

```diff
@@ -28,11 +28,36 @@ spec:
         command:
         - /app/backplane-operator
         env:
+        - name: OPERAND_IMAGE_BACKPLANE_MUST_GATHER
```

Comment:
Remove these env variables from this file before committing, and rerun `make bundle`